test/specs/lib.davIdIcon.spec.js

```diff
@@ -17,4 +17,18 @@ describe('getDavIdIconUrl()', () => {
       return false;  
     }
   };
+
+  test('Return contains the davID within it options',() => {     
+    expect(       
+      checkIfReturnDavId(getDavIdIconUrl(dummyDavId))     
+    ).toBeTruthy();   
+  });    
+  
+  const checkIfReturnDavId = (url) => {     
+    if (url.toLowerCase().indexOf(dummyDavId) >= 0) {       
```

Comment:
The real davID will contain both uppercase and lowercase letters. Please don't cast it to lowercase.

Reply:
Additionally, you can replace the indexOf function with ES2015 `string.prototype.includes` which results in much more readable code.